Source/common/SNTPostflightResult.h

```diff
@@ -0,0 +1,38 @@
+/// Copyright 2025 North Pole Security, Inc.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#include "Source/common/SNTCommonEnums.h"
+
+@class SNTSyncState;
+
+@interface SNTPostflightResult : NSObject <NSSecureCoding>
```

Comment:
I'm not a fan of this being so closely tied to "postflight"; this class is ultimately a way to update many settings at once